docs/pages/more/create-expo.mdx

```diff
@@ -63,13 +63,13 @@ Running `create-expo-app` with a [Node Package Manager](#node-package-managers-s
-| Default                 | Default template. Designed to build multi-screen apps. Includes recommended tools such as Expo CLI, Expo Router library and TypeScript configuration enabled. Suitable for most apps. |
-| Blank                   | Installs minimum required npm dependencies without configuring navigation.                                                                                                            |
-| Blank (TypeScript)      | A Blank template with TypeScript enabled.                                                                                                                                             |
-| Navigation (TypeScript) | Installs and configures file-based routing with TypeScript enabled.                                                                                                                   |
-| Blank (Bare)            | A Blank template with native directories (**android** and **ios**) generated. Runs [`npx expo prebuild`](/workflow/prebuild/) during the setup.                                       |
+| Template                                                                                              | Description                                                                                                                                                                           |
+| ----------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Default](https://github.com/expo/expo/tree/main/templates/expo-template-default)                     | Default template. Designed to build multi-screen apps. Includes recommended tools such as Expo CLI, Expo Router library and TypeScript configuration enabled. Suitable for most apps. |
+| [Blank](https://github.com/expo/expo/tree/main/templates/expo-template-blank)                         | Installs minimum required npm dependencies without configuring navigation.                                                                                                            |
+| [Blank (TypeScript)](https://github.com/expo/expo/tree/main/templates/expo-template-blank-typescript) | A Blank template with TypeScript enabled.                                                                                                                                             |
+| [ Navigation (TypeScript)](https://github.com/expo/expo/tree/main/templates/expo-template-tabs)       | Installs and configures file-based routing with TypeScript enabled.                                                                                                                   |
```

Comment:
```suggestion
| [Navigation (TypeScript)](https://github.com/expo/expo/tree/main/templates/expo-template-tabs)        | Installs and configures file-based routing with TypeScript enabled.                                                                                                                   |
```

Whitespace nit.

Reply:
Interesting. Prettier config didn't catch this on my system 😄